### PR TITLE
Modification to the user-page.html so that the message form displays on every user page

### PR DESCRIPTION
**Pull request by stanleynwakamma**
Modified the user-page.html, specifically the `showMessageFormIfViewingSelf()` function of the user-page-loader.js  file to enable its logic to show the form as long as the user is logged in, even if they aren't viewing their own page. With this change, the message form displays on every user page, not just on a user's own page. But please note that this doesn't actually change the behavior of the server

### CODE REVIEW

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -30,21 +30,23 @@ function setPageTitle() {
...
   fetch('/login-status')
       .then((response) => {
         return response.json();
       })
       .then((loginStatus) => {
-        if (loginStatus.isLoggedIn &&
-            loginStatus.username == parameterUsername) {
+        if (loginStatus.isLoggedIn) {
           const messageForm = document.getElementById('message-form');
+        messageForm.action = '/messages? recipient=' + parameterUsername;
```
Comment (kyrabusser) [inline src/main/webapp/js/user-page-loader.js]: I don't think there should be a space within '/messages?recipient='

**Review by kyrabusser — COMMENTED** [review]
Nice work, Stanley! I'm so excited to see your whole feature coming together! :+1: 

Can you take a screenshot of what this looks like in your localhost:8080 devserver?

Right now when I go to a different user's page (like @hengelman's: https://sp19-codeu-10-7182.appspot.com/user-page.html?user=hengel@umich.edu), there's no way for me to enter a new message. Is this different with your change?

Can you also try out the new message posting and check that it adds the message successfully? Check your Database Admin page (http://localhost:8080/_ah/admin) to see if all the data you expect is there.

Sorry if this seems like a lot of extra steps, but we want to make sure that every change makes sense within the final product as a whole. Let me know if you have any questions!

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (stanleynwakamma) [reply src/main/webapp/js/user-page-loader.js]: Updated the change in a new commit. Thanks for letting me know 😄

**Comment by stanleynwakamma** [comment]
> Nice work, Stanley! I'm so excited to see your whole feature coming together! 👍
> 
> Can you take a screenshot of what this looks like in your localhost:8080 devserver?
> 
> Right now when I go to a different user's page (like @hengelman's: https://sp19-codeu-10-7182.appspot.com/user-page.html?user=hengel@umich.edu), there's no way for me to enter a new message. Is this different with your change?
> 
> Can you also try out the new message posting and check that it adds the message successfully? Check your Database Admin page (http://localhost:8080/_ah/admin) to see if all the data you expect is there.
> 
> Sorry if this seems like a lot of extra steps, but we want to make sure that every change makes sense within the final product as a whole. Let me know if you have any questions!

Thank you so much @kyrabusser for the updates. I have attached a screenshot of the messaging from localhost:8080 
You may not see the new changes when you visit `https://sp19-codeu-10-7182.appspot.com/user-page.html?user` because we haven't deployed to app engine yet; but the changes show in the localhost server. 
This pull request is just a modification of the `user-page.html`, which is the step 5 of the project, whcih doesn't actually change the behavior of the server. My next pull request will have code that changes the behavior (i.e. step 6 of the project). But please note that the screenshots attached here are the final results of the step 6. 
I have also attached a screenshot from http://localhost:8080/_ah/admin  Everything seems pretty good there. Please let me know of any concerns. Thanks 😁 
user page 1
user page 2
user-page
window snap

**Review by kyrabusser — APPROVED** [review]
Awesome! 🎉 🎉 🎉 
Thanks for doing the testing and taking screenshots! 👍

**Pull request merged**